source/reference/operator/aggregation/dateFromParts.txt

```diff
@@ -0,0 +1,184 @@
+
+      * - ``isoYear``
+        - Required if not using ``year``
+        - ISO Week Date Year.
+        - ``0``-``9999``
+		
+      * - ``month``
+        - Optional
+        - Month. Defaults to ``1``.
+        - ``1``-``12``
+		
+      * - ``isoWeekYear``
+        - Optional
+        - Week of year. Defaults to ``1``.
+        - ``1``-``53``
+
+      * - ``day``
+        - Optional
+        - Day of month. Defaults to ``1``.
+        - ``1``-``31``
+
+      * - ``isoDayOfWeek``
+        - Optional
+        - Day of week (Monday ``1`` - Sunday ``7``). Defaults to ``1``.
+        - ``1``-``7``
+
+      * - ``hour``
+        - Optional
+        - Hour. Defaults to ``0``.
+        - ``0``-``23``
+
+      * - ``minute``
+        - Optional
+        - Minute. Defaults to ``0``.
+        - ``0``-``59``
+
+      * - ``second``
+        - Optional
+        - Second. Defaults to ``0``.
+        - ``0``-``59``
+
+      * - ``millisecond``
+        - Optional
+        - Millisecond. Defaults to ``0``.
+        - ``0``-``999``
+
+      * - ``timezone``
+        - Optional
+        - ``<timezone>`` can be any :ref:`expression
+          <aggregation-expressions>` that evaluates to a string whose
+          value is either:
+
+          - an `Olson Timezone Identifier
+            <https://en.wikipedia.org/wiki/List_of_tz_database_time_zones>`_, 
+            such as ``"Europe/London"`` or ``"America/New_York"``, or
+
+          - a UTC offset in the form:
+
+            - ``+/-[hh]:[mm]``, e.g. ``"+04:45"``, or
+
+            - ``+/-[hh][mm]``, e.g. ``"-0530"``, or
+
+            - ``+/-[hh]``, e.g. ``"+03"``.
+
+          For more information on expressions, see
+          :ref:`aggregation-expressions`.
+        - Timezone expression
+		
+   .. important::
+       You cannot combine the use of calendar dates and ISO week date
+       fields when constructing your :expression:`$dateFromParts` input
+       document.
+
+Behavior
+--------
+
+.. include:: /includes/fact-olson-tz-behavior.rst
+
+Example
+-------
+The following aggregation uses :expression:`$dateFromParts` to
+construct three date objects from the provided input fields:
+
+.. cssclass:: copyable-code
+.. code-block:: javascript
+
+   db.sales.aggregate([
+   {
+      $project: {
+         date: {
+            $dateFromParts: {
+               'year' : 2017, 'month' : 2, 'day': 8, 'hour' : 12
+            }
+         },
+         date_iso: {
+            $dateFromParts: {
+               'isoYear' : 2017, 'isoWeekYear' : 6, 'isoDayOfWeek' : 3, 'hour' : 12
+            }
+         },
+         date_timezone: {
+            $dateFromParts: {
+               'year' : 2016, 'month' : 12, 'day' : 31, 'hour' : 23,
+               'minute' : 46, 'second' : 12, 'timezone' : 'America/New_York'
+            }
+         }
+      }
+   }])
+
+The operation returns the following result for each document in the
+``sales`` collection.
+
+.. code-block:: javascript
+
+   {
+     "_id" : ObjectId("<id>"),
```

Comment:
Just FYI -- you inserted a document in the sales collection with an _id of 1.
We probably should be consistent and return that value.

Reply:
True, for what it's worth my thinking was that because that document was inserted as part of the Behavior section we might want to consider a separate entity / example.

What you're saying makes sense though, so I'll update.